src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java

```diff
@@ -1008,7 +1008,7 @@ else if(model instanceof AcogReticle) {
                                 }, 250, 0))
 
                         .build())
-                .withSpawnEntityDamage(5.4f)
+                .withSpawnEntityDamage(6.4f)
```

Comment:
For the fire rate it as this seems a bit high

Reply:
It uses 7.62x35mm bullets, so its power should be greater, which is not enough to affect balance as people prioritize using VSS